.github/workflows/ci.yml

```
@@ -0,0 +1,374 @@
+jobs:
+  # Job 1: Basic validation (fast feedback)
+  validate:
+    name: Code Validation
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      
+    - name: Setup Rust toolchain
```

actionlint (1.7.7)

step must run script with "run" section or run action with "uses" section

(syntax-check)